docs/usage/shoot_serviceaccounts.md

```diff
@@ -56,15 +56,19 @@ Given that the field just specifies the upper bound, end-users can still use low
 
 ## Managed Service Account Issuer
 
+Gardener also provides a way to manage the service account issuer of a shoot cluster as well as serving its OIDC discovery documents from a centrally managed server called [Gardener Discovery Server](https://github.com/gardener/gardener-discovery-server).
+This ability removes the need for changing the `.spec.kubernetes.kubeAPIServer.serviceAccountConfig.issuer` and exposing it separately.
+
 ### Prerequisites
 
-Gardener also provides a way to manage the service account issuer of a shoot cluster as well as serving its OIDC discovery documents from a centrally managed server called [Gardener Discovery Server](https://github.com/gardener/gardener-discovery-server). This ability removes the need for changing the `.spec.kubernetes.kubeAPIServer.serviceAccountConfig.issuer` and exposing it separately. The feature has the following prerequisites:
+⚠️ Note: The following prerequisites are responsibility of the Gardener Administrators and are not something that end users can configure by themselves.
+If uncertain that these requirements are met, please contact your Gardener Administrator.
```

Comment:
Nit: You could use the [new Github Notes syntax](https://github.com/orgs/community/discussions/16925):

```suggestion
> [!WARNING]
> The following prerequisites are the responsibility of the Gardener Administrators and are not something that end users can configure by themselves.
> If uncertain that these requirements are met, please contact your Gardener Administrator.
```